--- blueproximity-1.4.0.orig/proximity.py
+++ blueproximity-1.4.0/proximity.py
@@ -28,7 +28,7 @@ APP_NAME = "blueproximity"
 # This value gives us the base directory for language files and icons.
 # Set this value to './' for local folder version
 # or, for instance, to '/usr/share/blueproximity/' for packaged version
-dist_path = './'
+dist_path = '/usr/share/blueproximity/'
 
 # Translation stuff
 import gettext
